### ISSUE TITLE
Add strftime conversion into output.file.hls

### ISSUE THREAD

**Issue by cdgraff** [savonet/liquidsoap#2593]
To build long term archive of TS file I'll like to have the option to create subfolders to archive the TS file more organized.

I'm using that when run with `output.file` and will be great support the same into `output.file.hls`

```
(unlabeled) (of type {string}): Filename where to output the stream. Some strftime conversion specifiers are available: %SMHdmY. You can also use $(..) interpolation notation for metadata.
```

**Comment by smimram** [comment]
I have just added `time.string` in bb3e529283a5abba84fcf2368b03322159802905. This means that you should now be able to do something like
```
output.file(%wav, {string.time("file%H%M%S.wav")}, s)
```

**Comment by toots** [comment]
This is a great solution @smimram we should generalize this to all operators that previously had builtin `strftime`..

**Comment by toots** [comment]
This is fixed in [6df0185](https://github.com/savonet/liquidsoap/commit/6df0185bb7acc7034c1ce4b636a2d505caaf1c11). The reason for the removal is that it makes the API more predictable. Instead of having some operators (which? why these? why not all?) that support the conversion, one can now use the same solution and patterns (string getters) and a well-defined function `time.string` to make this explict. This also allows (unlikely) cases where the user actually does want to name their file `"file%H%M%S.wav"` 😅

Next on this trend would be the implicit interpolation syntax. However, this one is more tricky to replace as this requires to pass the metadata arguments to the string generator.
